components/python/ptyprocess/history

```diff
@@ -1 +1,2 @@
+library/python/ptyprocess-27@0.6.0-2020.0.1.4
 library/python/ptyprocess-34@0.6.0,5.11-2020.0.1.1
```

Comment:
,5.11 is missing here (the 2nd has it)

Reply:
I think we concluded that short fmris are supported since pkg was modified?